spec/rmagick/image/compare_channel_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe Magick::Image, "#compare_channel" do
+  before do
+    @img = Magick::Image.new(20, 20)
+  end
+
+  it "works" do
+    img1 = Magick::Image.read(IMAGES_DIR + '/Button_0.gif').first
```

Comment:
Not for this PR but maybe we could put `IMAGES_DIR + '/Button_0.gif'` into a `TestImages` class and add `Button0` field/property?

Reply:
Yeah, I think we can come up with a better way of handling this.